frontend/src/app/globals.css

```
@@ -1,911 +1,230 @@
-  background: none;
-  color: inherit;
-  padding: 0;
-  font-size: inherit;
-}
-
-input, select, textarea { color-scheme: light; }
-
-/* ── Layout shells ──────────────────────────────────────────────── */
-.panel {
-  background: var(--bg-panel);
+  --font-display: 'Playfair Display', 'Spectral', Georgia, serif;
```

Stylelint (17.7.0)

[error] Expected "Georgia" to be "georgia" (value-keyword-case)

(value-keyword-case)